### DISCUSSION TITLE
cannot insert using slick

### DISCUSSION THREAD

**Discussion by zuliandri2** [slick/slick#2628] [Questions] [answered]
hi
I am learning slick and play framework, I have a problem with slick, I cannot insert data into database
how to insert data using slick?
here my code
![image](https://user-images.githubusercontent.com/10961433/210078087-061d94b6-49ad-42a9-9c71-feff1e8cf9d1.png)

this is my config
![image](https://user-images.githubusercontent.com/10961433/210078038-98e3ed20-e2c1-4a73-bf88-2b4b394d567d.png)
 when I tried to execution those code, it wasn't noticed
 
![image](https://user-images.githubusercontent.com/10961433/210078273-f816c15b-e62e-43d7-a3d9-da36b1a52592.png)

when i tried to execution this code, it's worked
![image](https://user-images.githubusercontent.com/10961433/210078353-657cba3c-7dd5-42da-8e5c-e9d23e5fe3a6.png)

this is my dependency
![image](https://user-images.githubusercontent.com/10961433/210078478-0e67dda2-c094-450c-b52c-c562b2276f39.png)

**Comment by nafg** [comment, marked as answer]
That is because the future runs in the background and the main thread exits before it gets a chance to finish.

See https://stackoverflow.com/a/23548889/333643

**Reply by zuliandri2** [reply]
@nafg thanks, finaly I am getting the error
